src/test/java/sender/SenderDataTest.java

```diff
@@ -0,0 +1,33 @@
+package sender;
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import sender.SederDataService;
+
+public class SenderDataTest {
+	
+	SederDataService senderData = new SederDataService();
+
+	
+	@Test
+	public void getBMSDataTest() {
```

Comment:
The crucial functionality of the sender is the output format. Without a test for the output format, a maintainer might alter it and realize only when integrating with the receiver.
It would take a lot less time if the output format is checked with a unit test.